CMakeLists.txt

```diff
@@ -0,0 +1,96 @@
+set(PYTHON_SCRIPT_DIR python-scripts)
+macro(add_python_target target)
+    foreach (file ${target})
+        get_filename_component(file_name ${file} NAME)
+        configure_file(${file} ${PYTHON_SCRIPT_DIR}/${file_name} COPYONLY)
+    endforeach ()
+endmacro()
+
+cmake_minimum_required(VERSION 3.1.0)
+
+project(minotaur-cpp)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_PREFIX_PATH X:\\Qt\\5.8\\mingw53_32)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${Qt5Widgets_EXECUTABLE_COMPILE_FLAGS} -std=c++11")
+
+find_package(PythonLibs REQUIRED)
+find_package(Qt5Widgets REQUIRED)
+find_package(Qt5Core REQUIRED)
+find_package(Qt5Gui REQUIRED)
+
+set(SOURCE_FILES code/main.cpp
+
```

Comment:
use GLOB_RECURSE to enable wildcards, so e.g. change this to:

`file(GLOB_RECURSE SOURCE_FILES ${minotaur_cpp_SOURCE_DIR}/code/*.cpp ${minotaur_cpp_SOURCE_DIR}/third-party/*.cpp)`, or something along those lines.